packages/cardano-services/package.json

```diff
@@ -17,25 +17,27 @@
   "license": "MPL-2.0",
   "scripts": {
     "build": "tsc --build ./src && bash ./copy-openApi.sh",
-    "tscNoEmit": "shx echo typescript --noEmit command not implemented yet",
```

Comment:
This can be committed separately as a `chore: script in alphabetical order in package.json file`.